apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/transaction/types/cc/CCBidOrderPlacementTransactionType.java

```diff
@@ -61,11 +61,18 @@ public ColoredCoinsBidOrderPlacement parseAttachment(JSONObject attachmentData)
     @Override
     public boolean applyAttachmentUnconfirmed(Transaction transaction, Account senderAccount) {
         ColoredCoinsBidOrderPlacement attachment = (ColoredCoinsBidOrderPlacement) transaction.getAttachment();
-        if (senderAccount.getUnconfirmedBalanceATM() >= Math.multiplyExact(attachment.getQuantityATU(), attachment.getPriceATM())) {
-            getAccountService().addToUnconfirmedBalanceATM(senderAccount, getLedgerEvent(), transaction.getId(), -Math.multiplyExact(attachment.getQuantityATU(), attachment.getPriceATM()));
-            return true;
+        try
+        {
+            if (senderAccount.getUnconfirmedBalanceATM() >= Math.multiplyExact(attachment.getQuantityATU(), attachment.getPriceATM())) {
+                getAccountService().addToUnconfirmedBalanceATM(senderAccount, getLedgerEvent(), transaction.getId(), -Math.multiplyExact(attachment.getQuantityATU(), attachment.getPriceATM()));
+                return true;
+            }
+            return false;
+        }
+        catch (java.lang.ArithmeticException e)
+        {
```

Comment:
Add logging here.
put
`@Slf4j`
on top of class,
put logging into catch block:
`log.error("Error: attachment = {}", attachment);`